test/kuttl/tests/cinder_scale/02-assert.yaml

```diff
@@ -0,0 +1,205 @@
+        - /usr/bin/dumb-init
+        imagePullPolicy: IfNotPresent
+        resources: {}
+        securityContext:
+          runAsUser: 0
+        terminationMessagePath: /dev/termination-log
+        terminationMessagePolicy: File
+        volumeMounts:
+        - mountPath: /var/log/cinder
+          name: logs
+      - args:
+        - -c
+        - /usr/local/bin/kolla_set_configs && /usr/local/bin/kolla_start
+        command:
+        - /bin/bash
+        imagePullPolicy: IfNotPresent
+        livenessProbe:
+          failureThreshold: 3
+          httpGet:
+            path: /healthcheck
+            port: 8776
+            scheme: HTTP
+          initialDelaySeconds: 5
+          periodSeconds: 3
+          successThreshold: 1
+          timeoutSeconds: 5
+        name: cinder-api
+        readinessProbe:
+          failureThreshold: 3
+          httpGet:
+            path: /healthcheck
+            port: 8776
+            scheme: HTTP
+          initialDelaySeconds: 5
+          periodSeconds: 5
+          successThreshold: 1
+          timeoutSeconds: 5
+        resources: {}
+        securityContext:
+          runAsUser: 0
+        terminationMessagePath: /dev/termination-log
+        terminationMessagePolicy: File
+        volumeMounts:
+        - mountPath: /etc/machine-id
+          name: etc-machine-id
+          readOnly: true
+        - mountPath: /etc/localtime
+          name: etc-localtime
+          readOnly: true
+        - mountPath: /usr/local/bin/container-scripts
+          name: scripts
+          readOnly: true
+        - mountPath: /var/lib/config-data/merged
+          name: config-data
+          readOnly: true
+        - mountPath: /etc/my.cnf
+          name: config-data
+          readOnly: true
+          subPath: my.cnf
+        - mountPath: /etc/cinder/cinder.conf.d
+          name: config-data-custom
+          readOnly: true
+        - mountPath: /var/lib/kolla/config_files/config.json
+          name: config-data
+          readOnly: true
+          subPath: cinder-api-config.json
+        - mountPath: /var/log/cinder
+          name: logs
+      dnsPolicy: ClusterFirst
+      restartPolicy: Always
+      serviceAccount: cinder-cinder
+      serviceAccountName: cinder-cinder
+      volumes:
+      - hostPath:
+          path: /etc/machine-id
+          type: ""
+        name: etc-machine-id
+      - hostPath:
+          path: /etc/localtime
+          type: ""
+        name: etc-localtime
+      - secret:
+          defaultMode: 493
+          secretName: cinder-scripts
+        name: scripts
+      - secret:
+          defaultMode: 420
+          secretName: cinder-config-data
+        name: config-data
+      - secret:
+          defaultMode: 420
+          secretName: cinder-api-config-data
+        name: config-data-custom
+      - emptyDir: {}
+        name: logs
+  updateStrategy:
+    rollingUpdate:
+      partition: 0
+    type: RollingUpdate
+status:
+  availableReplicas: 3
+  replicas: 3
```

Comment:
I think this is the essence of the test. With this in mind, it would be good if you could trim the "assert" data down to the minimum, and not repeat everything that's already tested in the 01-assert.yaml (which is a symlink to ../../common/assert_sample_deployment.yaml.

The reason I'd like to see this 02-assert.yaml just test the replicas is it reduces the burden of maintaining all the assert files when something fundamental changes. We should be able to rely on 01-assert.yaml testing a lot of the deployment, and subsequent 02..0N-assert.yaml should just need to test the deltas.

Reply:
Thanks for the feedback, I will look into it.

Reply:
Ack, the trimmed file version worked in my setup earlier today. I will update the PR shortly. Thanks for the follow up.